### PR TITLE
feat: apply new design(default theme)

### PR DESCRIPTION
**Pull request by js87zz**
### Please check if the PR fulfills these requirements
- [x] It's the right issue type on the title
- [ ] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [x] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes/features)
- [ ] Docs have been added/updated (for bug fixes/features)
- [ ] It does not introduce a breaking change or has a description of the breaking change

### Description
applied new design for `v3.0`(default theme)

* main image
스크린샷 2021-04-19 오전 10 39 25

* table popup
스크린샷 2021-04-19 오전 10 36 34

* dropdown toolbar
스크린샷 2021-04-19 오전 10 36 47

* image popup
스크린샷 2021-04-19 오전 10 37 12

---
Thank you for your contribution to TOAST UI product. 🎉 😘 ✨

### CODE REVIEW

**Review by dotaitch — APPROVED** [review]
리뷰 완료합니다.

File: apps/editor/src/css/editor.css [hunk 1]
```diff
@@ -394,119 +440,152 @@
...
+  left: 0;
+  border: 1px solid #00a9ff;
+  background: rgba(0, 169, 255, 0.1);
+  z-index: 999;
+}
+
+.toastui-editor-popup-add-table .toastui-editor-table-description {
+  margin: 5px 0 0 0;
+  text-align: center;
+  color: #333333
```
Comment (lja1018) [inline apps/editor/src/css/editor.css:588]: `#333`이 될 것 같습니다.

File: apps/editor/src/css/editor.css [hunk 2]
```diff
@@ -394,119 +440,152 @@
...
+  position: absolute;
+  top: 0;
+  left: 0;
+  border: 1px solid #00a9ff;
+  background: rgba(0, 169, 255, 0.1);
+  z-index: 999;
+}
+
+.toastui-editor-popup-add-table .toastui-editor-table-description {
+  margin: 5px 0 0 0;
```
Comment (lja1018) [inline apps/editor/src/css/editor.css:586]: `5px 0 0`으로 단축해도 될 것 같습니다.

File: apps/editor/src/css/editor.css [hunk 3]
```diff
@@ -566,22 +646,124 @@
   font-size: 14px;
 }
 
+/* table context menu */
+.toastui-editor-context-menu {
+  position: absolute;
+  width: auto;
+  min-width: 197px;
+  color: #333333;
```
Comment (lja1018) [inline apps/editor/src/css/editor.css:654]: `#333`이 될 것 같습니다.

File: apps/editor/src/css/editor.css [hunk 4]
```diff
@@ -566,22 +646,124 @@
...
+  border-radius: 2px;
+  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.08);
+  border: solid 1px #dadde6;
```
Comment (lja1018) [inline apps/editor/src/css/editor.css:657]: 줄여쓰겠습니다... `1px solid #dadde6`

File: apps/editor/src/css/editor.css [hunk 5]
```diff
@@ -566,22 +646,124 @@
...
+}
+
+.toastui-editor-context-menu li.disabled {
+  color: #c9ccd5;
+}
+
 .toastui-editor-tooltip {
   position: absolute;
-  background-color: #222;
+  background-color: #444444;
```
Comment (lja1018) [inline apps/editor/src/css/editor.css:751]: `#444`

File: apps/editor/src/css/editor.css [hunk 6]
```diff
@@ -566,22 +646,124 @@
...
+    'Nanum Barun Gothic', '맑은고딕', 'Malgun Gothic', sans-serif;
 }
 
 .toastui-editor-tooltip .arrow {
   content: '';
   display: inline-block;
   width: 10px;
   height: 10px;
-  background-color: #222;
+  background-color: #444444;
```
Comment (lja1018) [inline apps/editor/src/css/editor.css:766]: `#444`

File: apps/editor/src/css/editor.css [hunk 7]
```diff
@@ -606,146 +787,85 @@
...
   .toastui-editor-toolbar-icons {
-    background: url('../img/toastui-editor-2x.png');
-    background-size: 218px 188px;
-    display: inline-block;
+    background: url('../img/toastui-editor-2x.png') no-repeat;
+    background-size: 466px 146px;
+  }
+  .toastui-editor-context-menu span::before {
+    background: url('../img/toastui-editor-2x.png') no-repeat;
+    background-size: 466px 146px;
```
Comment (lja1018) [inline apps/editor/src/css/editor.css:795]:
```
.toastui-editor-toolbar-icons,
.toastui-editor-context-menu span::before {
```
로 합쳐도 되지 않을까요?

**Review by lja1018 — APPROVED** [review]
리뷰완료합니다.

**Review by shiren — APPROVED** [review]
수고하셨습니다.

**Review by jungeun-cho — APPROVED** [review]
수고하셨어요. 리뷰완료했습니다.

**Pull request merged**